src/ui/ResultsPerPage/ResultsPerPage.ts

```diff
@@ -233,12 +233,15 @@ export class ResultsPerPage extends Component {
     for (var i = 0; i < numResultsList.length; i++) {
       const listItem = $$('li', {
         className: 'coveo-results-per-page-list-item',
+        role: 'radio',
         tabindex: 0
       }).el;
       const resultsPerPage = numResultsList[i];
-      if (resultsPerPage === this.currentResultsPerPage) {
+      const isActive = resultsPerPage === this.currentResultsPerPage;
+      if (isActive) {
         $$(listItem).addClass('coveo-active');
       }
+      listItem.setAttribute('aria-checked', isActive.toString());
```

Comment:
Why are we using `aria-checked` here vs. `aria-pressed` above? It looks like `aria-checked` is primarly used to describe the state of checkboxs:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role

Reply:
Because `Pager` is a `navigation` component with toggle-able buttons, while `ResultsPerPage` is a `radiogroup` with `radio` elements.

https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html